### ISSUE TITLE
[TRB] (Lisa Elbert) Accessibility Testing & Implementation

### ISSUE THREAD

**Issue by siddheshraze** [Smithsonian/ForestGEO#211]
There are currently no accessibility standards in place for this application. A11y standards should be maintained and tested throughout the app.

**Comment by justinpac** [comment]
Need to access ServiceNow - need to reach out to Lauren to submit the app for a scan.

**Comment by justinpac** [comment]
Lisa Elbert is supposed to do the scan - we've given them sign in info. Waiting to hear back from them since June 6th.

**Comment by justinpac** [comment]
We now have results from Lisa. Should take about three days fix.

**Comment by justinpac** [comment]
Should get these fixes in by August 7th.

**Comment by justinpac** [comment]
Accessibility changes are in, now we will email Lisa to confirm it is good.

**Comment by justinpac** [comment]
On Friday Aug 15 Lisa sent a mail saying the site is again being reviewed. We followed today to ask for ETA.

**Comment by justinpac** [comment]
Last we heard from them was on August 22nd. Following up over email today to figure out the status.

**Comment by siddheshraze** [comment]
Feedback received from latest set of scans - some remaining errors from WCAG 2.0 were found. Updating system and contacting Lauren to reopen an accessibility testing request.

**Comment by justinpac** [comment]
Will apply accessibility changes directly to main branch and merge them.

**Comment by justinpac** [comment]
Applied changes to the live site. Lauren was emailed to resubmit.

**Comment by justinpac** [comment]
We're waiting on Lauren, but blocked due to govt shutdown.
